manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/MergeTest.scala

```diff
@@ -228,11 +245,55 @@ return movie""",
         """match (charlie:Person {name:'Charlie Sheen'}), (oliver:Person {name:'Oliver Stone'})
 merge (charlie)-[r:KNOWS]-(oliver)
 return r""",
-      optionalResultExplanation = "Assume that Charlie Sheen and Oliver Stone do not know each other then " +
+      optionalResultExplanation = "As 'Charlie Sheen' and 'Oliver Stone' do not know each other, " +
         "this +MERGE+ query will create a +:KNOWS+ relationship between them. " +
         "The direction of the created relationship is arbitrary.",
       assertions = (p) => assertStats(p, relationshipsCreated = 1)
     )
   }
 
+  @Test def merge_on_a_relationship_between_two_existing_nodes() {
+    testQuery(
+      title = "Merge on a relationship between two existing nodes",
+      text =
+        """+MERGE+ can be used in conjunction with preceding +MATCH+ and +MERGE+ clauses to create a relationship
+between two bound nodes 'm' and 'n', where 'm' is returned by +MATCH+ and 'n' is created or matched by the earlier +MERGE+.""",
+      queryText =
+        """match (person:Person)
+merge (city:City {name: person.bornIn})
+merge (person)-[r:BORN_IN]->(city)
+return person, r, city""".stripMargin,
+      optionalResultExplanation =
+        """This builds on the example from the 'Merge single node derived from existing node property' section above
+(in <<query-merge-node-derived>>). The second +MERGE+ creates a +BORN_IN+ relationship between each person and a city
+corresponding to the value of the person’s 'bornIn' property. 'Charlie Sheen', 'Rob Reiner' and 'Oliver Stone' all have
+a +BORN_IN+ relationship to the 'same' +City+ node ('New York').""",
+      assertions = (p) => assertStats(p, nodesCreated = 3, propertiesSet = 3, labelsAdded = 3, relationshipsCreated = 5)
+    )
+  }
+
+  @Test def merge_on_a_relationship_between_an_existing_node_and_a_merged_node_derived_from_a_node_property() {
+    testQuery(
+      title = "Merge on a relationship between an existing node and a merged node derived from a node property",
+      text =
+        """+MERGE+ can be used to create simultaneously both a new node 'n' and a relationship between a bound node 'm'
+and 'n'.""".stripMargin,
+      queryText =
+        """match (person:Person)
+merge (person)-[r:HAS_CHAUFFEUR]->(chauffeur:Chauffeur {name: person.chauffeurName})
+return person, r, chauffeur""".stripMargin,
```

Comment:
Same as for the above (all of the chauffeur data is shown so that the reader can see that it is two different 'John Brown' nodes that are created). I have removed the relationship.